src/components/legend.ts

```diff
@@ -29,8 +29,8 @@ export module Components {
     /**
      * Creates a Legend.
      *
-     * The Legend consists of a series of entries, each with a color and label taken from the `scale`.
-     * The entries will be displayed in the order of the `scale` domain.
+     * The Legend consists of a series of entries, each with a color and label taken from the Scales.Color.
+     * By defaul, the entries will be in the same order as the Scale's domain.
```

Comment:
I don't think the entries description is necessary and can be moved to the comparator